src/libs/OptionsListUtils.ts

```diff
@@ -1302,6 +1338,8 @@ function getValidOptions(
         // User to invite is generated by the search input of a user.
         // As this function isn't concerned with any search input yet, this is null (will be set when using filterOptions).
         userToInvite: null,
+        recentWorkspaceChats,
```

Comment:
I think we should build an all workspace options instead of recent workspace chats in this case, as we cannot search with the existing list from settings/workspaces
 
![Screenshot 2024-12-14 at 12 20 49](https://github.com/user-attachments/assets/61e50fd5-c03e-49b2-9272-477de6e2e0d0)


![Screenshot 2024-12-14 at 12 28 04](https://github.com/user-attachments/assets/1f26eed5-7b52-4163-b971-b256b981b691)

Reply:
So basically your suggestion would be to get rid of recent workspaces and instead of that have all workspaces list with the default workspace on top?

Reply:
Yes, based on [the OP](https://github.com/Expensify/App/issues/52981#issue-2684450581) `Add a section for ‘Workspaces’ at the top (if you’re a member of a workspace).`